examples/operator/101_initial_cluster.yaml

```diff
@@ -27,9 +27,9 @@ spec:
       resources:
         requests:
           cpu: 100m
-          memory: 256Mi
+          memory: 128Mi
         limits:
-          memory: 256Mi
+          memory: 512Mi
```

Comment:
very minor: can you move limits above request , in order to be consistent with rest of the yaml file. will be easy to read. provided it does not break any thing.

Reply:
Sure! That lack of uniformity caught me at first too. I'll do that now, although I'll use requests,limits ordering as I think that's a more logical order for readers (does not matter in yaml parsing).